### PR TITLE
style action button  and active Nav Links

### PR DESCRIPTION
**Pull request by willyserge**
# what does this PR do?
 - animated the homepage action button
- style and set active state to Nav Links

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -1,46 +1,57 @@
...
+import { Link } from "gatsby"
+import styled from "styled-components"
+import "./index.scss"
+import Sidebar from "./Sidebar"
 
 const Logo = styled.img`
   width: 75px;
   height: 75px;
   padding-top: 30px;
-`;
```
Comment (eliemugenzi) [inline src/components/Header/index.jsx:12]: Why are you removing semi-colons?

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -1,46 +1,57 @@
...
+`
 
 const Header = () => {
-  const [open, setOpen] = useState(false);
-  const logoUrl = useSelector((state) => state.ui.logoUrl);
+  const [open, setOpen] = useState(false)
+  const logoUrl = useSelector(state => state.ui.logoUrl)
```
Comment (eliemugenzi) [inline src/components/Header/index.jsx:16]: Check if your ESLint is highlighting all of these issues.

File: src/components/Header/index.jsx (hunk 1 above)
Comment (willyserge) [reply src/components/Header/index.jsx:12]: might be an issue with prettier.

Comment (eliemugenzi) [reply src/components/Header/index.jsx:12]: Yeah, disable `Prettier` for now and let ESLint handle the rest. We'll reconfigure Prettier with ESLint later as we go.

**Pull request merged**